src/App.vue

```diff
@@ -1,14 +1,12 @@
 <script setup lang="ts">
-import { RouterLink, RouterView } from 'vue-router'
+  import { RouterLink, RouterView } from 'vue-router';
 </script>
-
-
 <template>
   <div id="app">
     <header>
       <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/todo-list">Go to App</RouterLink>
+        <RouterLink :to="{ name: 'home' }">Home</RouterLink>
+        <RouterLink :to="{ name: 'todolist' }">Go to App</RouterLink>
```

Comment:
change this to enums
```suggestion
        <RouterLink :to="{ name: RouteNames.HOME }">Home</RouterLink>
        <RouterLink :to="{ name: RouteNames.TODOLIST }">Go to App</RouterLink>
```